src/main/java/org/broadinstitute/hellbender/tools/recalibration/RecalUtils.java

```diff
@@ -190,26 +75,26 @@
     private static class CsvPrinter {
 
         private final PrintStream ps;
-        private final Covariate[] covariates;
+        private final StandardCovariateList covariates;
 
         /**
          * Constructs a printer redirected to an output file.
          * @param out the output file.
-         * @param c covariates to print out.
+         * @param covs covariates to print out.
          * @throws java.io.FileNotFoundException if the file could not be created anew.
          */
-        protected CsvPrinter(final File out, final Covariate ... c)
+        protected CsvPrinter(final File out, final StandardCovariateList covs)
```

Comment:
It's a little weird that we're now passing a `StandardCovariateList` around everywhere, given that this list is always the same. Would a singleton pattern perhaps be more appropriate here?